core/web/log_controller.go

```diff
@@ -60,6 +63,35 @@ func (cc *LogController) Patch(c *gin.Context) {
 		}
 	}
 
+	if request.ServiceName != "" && request.ServiceLevel != "" {
+		var level zapcore.Level
+		if err := level.UnmarshalText([]byte(request.ServiceLevel)); err != nil {
+			jsonAPIError(c, http.StatusInternalServerError, err)
+			return
+		}
+
+		if err := cc.App.SetServiceLogger(c.Request.Context(), request.ServiceName, level); err != nil {
+			jsonAPIError(c, http.StatusInternalServerError, err)
+			return
+		}
+
+		ll, err := cc.App.GetStore().Config.ServiceLogLevel(request.ServiceName)
+		if err != nil {
+			jsonAPIError(c, http.StatusInternalServerError, err)
+			return
+		}
+		response := &presenters.ServiceLevelLog{
```

Comment:
The endpoint should respond with a single object with all the data encompassing the resource. Here you are conditionally returning a `ServiceLevelLog` or a `LogResource`

Consider either splitting this into 2 separate endpoints, or combining the both responses as a single resource. I would lean towards splitting.

Reply:
Simplified this into one endpoint for now as it was seemingly confusing to have separate endpoints / responses. Set logic so we can pass in one service or multiple services / levels with a comma separated list and then render a standard "logConfig" back rather than separate resources / rendering

Let me know what you think about this and if it makes more sense to separate now